lisa/util/__init__.py

```diff
@@ -231,3 +232,20 @@ def is_valid_url(url: str, raise_error: bool = True) -> bool:
 
 def filter_ansi_escape(content: str) -> str:
     return __ansi_escape.sub("", content)
+
+
+def dump_files(os, environment) -> List[TextIOWrapper]:
```

Comment:
Make this method is more general using, not only for current purpose, that's why it should be in the `util/__init__.py`. The interface can be,
`def dump_file(file_name: Path, content: Any) -> None:`